_issues/bad_experience.md

```diff
@@ -6,18 +6,30 @@ categories: experience
   
 Q: Hillary has experience, but isn't it bad experience?  
   
-A: In the first Presidential debate, Donald Trump claimed that Hillary "has experience, but it's bad experience." This is utter nonsense. Hillary's record of public service spans four decades, starting with her first job out of law school. Here are just a few examples of [Hillary's many accomplishments](https://www.hillaryclinton.com/feed/seven-hillary-clintons-biggest-accomplishments/):
+A: In the first Presidential debate, Donald Trump claimed that Hillary "has experience, but it's bad experience." The words "bad experience" imply aa lack of success in her various roles. This is simply not true. Hillary's record of public service spans four decades, starting with her first job out of law school, and she has seen a [number of successes and accomplishments](https://www.hillaryclinton.com/feed/seven-hillary-clintons-biggest-accomplishments/) in that time:
```

Comment:
This doesn't really answer the question Trump was asking. Here's what he says:

“We have made so many bad deals during the last – so she’s got experience. I agree. But it’s bad, bad experience. Whether it’s the Iran deal you’re so in love with where we gave them $150 billion back. Whether it’s the Iran deal, whether it’s anything you can name – you almost can’t name a good deal." We should at the very minimum answer this question.